configuration/concepts/drugs_concepts.csv

```diff
@@ -311,4 +312,109 @@ b2727136-70f5-470b-81a2-26aeff3f346f,,,"Levonorgestrel, Ethinylestradiol","Levon
+56b34ea9-95f6-42ea-a582-fc513494602e,,,Tetracycline,Tetracycline,Drug,N/A,,,
+23df848e-435a-4f8a-b6bc-27235b98e018,,,Timolol,Timolol,Drug,N/A,,,
+d37b5fdb-50c3-40f2-906d-7c81bc22649b,,,Predisolone,Predisolone,Drug,N/A,,,
+e5a0d185-4c6a-42c8-a21e-e7436720d03c,,,Sodium hyaluronate & Oxymetazoline,Sodium hyaluronate & Oxymetazoline,Drug,N/A,,,
+8ebe6adf-72e3-495e-860f-c4a71ffc8016,,,Sodium chloride,Sodium chloride,Drug,N/A,,,
+3ea490ec-60e1-446e-83ca-6f5d1da8c2e6,,,Brimonidine tartrate,Brimonidine tartrate,Drug,N/A,,,
+b39042a0-345a-478c-8f8c-91286aa066d6,,,Betaxolol,Betaxolol,Drug,N/A,,,
+04c434fc-ab58-4a4f-84c5-18e6eaa08646,,,Brimonidine & timolol,Brimonidine & timolol,Drug,N/A,,,,
+3d0fed4d-1fd4-44a4-99fd-d59401067134,,,Dorzolamine & timolol,Dorzolamine & timolol,Drug,N/A,,,
+975c29db-ab3a-4c08-a3fb-829f79a6a7e8,,,Pilocarpine,Pilocarpine,Drug,N/A,,,
+32ed8cb1-bc73-44aa-934d-bf7626154af8,,,Cyclopentolate,Cyclopentolate,Drug,N/A,,,
+c5e3d595-76ae-4bf0-81d2-b3db54c36e6b,,,Propacaine,Propacaine,Drug,N/A,,,
+8d6744e1-4607-4cef-abd6-0827113e9cbe,,,Phenylephrine,Phenylephrine,Drug,N/A,,,
+e1b3ec78-175f-449b-a689-250641da6a05,,,Salbutamol & Iprotopium bromide,Salbutamol & Iprotopium bromide,Drug,N/A,,,
+b2841832-add7-4c6d-aa6e-425b97bb717d,,,Alcohol,Alcohol,Drug,N/A,,,
+66010c7d-dd21-4f83-b950-e2fff1ff6a73,,,Chlorhexidine,Chlorhexidine,Drug,N/A,,,
+cee47202-131e-46b9-8d23-3321cc03f750,,,Eucerin,Eucerin,Drug,N/A,,,
+f3ea42fb-f51f-4083-8b63-d787f8e991c5,,,Hemarrhoidal,Hemarrhoidal,Drug,N/A,,,
+73bbac06-6b84-4d1c-b961-9fd6437ba89f,,,Mineral oil,Mineral oil,Drug,N/A,,,
+dd850b07-d068-441b-a4e9-a806b4f8f004,,,Neomycin / Bacitracin,Neomycin / Bacitracin,Drug,N/A,,,
+5b1d3ede-3a08-4939-9b37-887bb63f68f9,,,Permanganate potassium,Permanganate potassium,Drug,N/A,,,
+6985c680-2c42-440a-bbbf-bedeeefe71e7,,,Petrolatum,Petrolatum,Drug,N/A,,,
+f7ae1d83-7687-45da-8327-8bf5532618e9,,,Povidone iodine,Povidone iodine,Drug,N/A,,,
+89e4c545-85af-4d9c-9e8c-c8e54f815cf0,,,Silver sulfadiazine,Silver sulfadiazine,Drug,N/A,,,
+407ff256-aa15-4dac-92c1-8173ba46232c,,,Benzathine PCN,Benzathine PCN,Drug,N/A,,,
+2bcf9bff-87d5-4abc-b95b-e212f6861dc5,,,Benzyl PCN,Benzyl PCN,Drug,N/A,,,
+aee293c2-1ca7-413e-b85a-50ae41bc5305,,,Benzyl PCN Procaine,Benzyl PCN Procaine,Drug,N/A,,,
+1605530b-1d7a-4a66-bb68-9be3c40d0c74,,,Cefalexin,Cefalexin,Drug,N/A,,,
+a93ad5bd-6f39-45d1-8fa7-d21c5bbc7ddd,,,Cefotaxime,Cefotaxime,Drug,N/A,,,
+404e784f-e4d8-427d-ad64-196348672168,,,Erythromycin,Erythromycin,Drug,N/A,,,
+9019b2e5-62fd-4238-b448-909a44f845ca,,,Gentamicin,Gentamicin,Drug,N/A,,,
+36d8bcf5-01f1-4c60-ac78-a7b5ccd31f2d,,,Quinine sulfate,Quinine sulfate,Drug,N/A,,,
+facb9b18-efad-4c4f-8855-0d20316a9326,,,Serum antitetatique,Serum antitetatique,Drug,N/A,,,
+41f47998-29ae-4f21-ad8c-32c50d0939f6,,,Vancomycin,Vancomycin,Drug,N/A,,,
+ab684df6-849b-42f7-84db-e4d974d3941c,,,Rifampicine & Isoniazide & Pyrazinamide,Rifampicine & Isoniazide & Pyrazinamide,Drug,N/A,,,

```

Comment:
in En: "Rifampicin & Isoniazide & Pyrazinamide"

Reply:
Done